scripts/aem.js

```diff
@@ -393,8 +393,8 @@ function decorateButtons(element) {
       const up = a.parentElement;
       const twoup = a.parentElement.parentElement;
       if (!a.querySelector('img')) {
-        if (up.childNodes.length === 1 && (up.tagName === 'P' || up.tagName === 'DIV')) {
-          a.className = 'button'; // default
+        if ((up.tagName === 'P' || up.tagName === 'DIV') && up.querySelectorAll('a').length >= 1) {
```

Comment:
We are discouraged from making changes to aem.js.  Can this be accomplished without making changes to this code?